### PR TITLE
Feature/dp 590 press continue button many times issue

### PR DESCRIPTION
**Pull request by JBaigGoaco**
To Prevent users from clicking multiple times on submit requests.
data-prevent-double-click="true" data-module="govuk-button"

https://design-system.service.gov.uk/components/button/#stop-users-from-accidentally-sending-information-more-than-once

image

### CODE REVIEW

**Review by maciej-goaco — APPROVED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationEmail.cshtml [hunk 1]
```diff
@@ -49,7 +49,7 @@
 
                 <button type="submit"
                         class="govuk-button"
-                        data-module="govuk-button">
+                        data-prevent-double-click="true" data-prevent-double-click="true" data-module="govuk-button">
```
Comment (jakzal) [inline Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationEmail.cshtml:52]: `data-prevent-double-click="true"` is doubled here.

File: Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationName.cshtml [hunk 2]
```diff
@@ -43,7 +43,7 @@
 
                 <button type="submit"
                         class="govuk-button"
-                        data-module="govuk-button">
+                        data-prevent-double-click="true" data-prevent-double-click="true" data-module="govuk-button">
```
Comment (jakzal) [inline Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationName.cshtml:46]: `data-prevent-double-click="true"` is doubled here.

File: Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationEmail.cshtml (hunk 1 above)
Comment (JBaigGoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationEmail.cshtml]: well spotted - all updated now

File: Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationName.cshtml (hunk 2 above)
Comment (JBaigGoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationName.cshtml]: well spotted - all updated now

**Review by jakzal — APPROVED** [review]

**Pull request merged**